ddp-workspace/projects/ddp-dsm-ui/src/app/participant-list/pipes/noData.pipe.ts

```diff
@@ -7,7 +7,7 @@ import {Pipe, PipeTransform} from '@angular/core';
 export class NoDataPipe implements PipeTransform {
   transform(value: any): string {
     const tempElement = document.createElement('div');
-    tempElement.innerHTML = value || '&nbsp;';
+    tempElement.innerHTML = (typeof value === 'number' ? String(value) : value) || '&nbsp;';
```

Comment:
What type the value is supposed to be?
If it is only **number** / **string** / **boolean** maybe using `value.toString()` is just enough.

Reply:
it can be anything

Reply:
including null and undefined